content/docs/guides/partner-billing.md

```diff
@@ -0,0 +1,350 @@
+<details>
+<summary>Response body</summary>
+<CodeBlock highlight="5,10,11,13,15,18-20,24,29,30,32,33,35-37,40,42">
+```json
+{
+  "projects": [
+    {
+      "project_id": "wispy-wind-94231251",
+      "period_id": "6fa781c3-fe37-45fa-9987-26a0d06edbd9",
+      "data_storage_bytes_hour": 6097554392,
+      "data_storage_bytes_hour_updated_at": "2023-11-08T19:07:53Z",
+      "synthetic_storage_size": 32616552,
+      "synthetic_storage_size_updated_at": "2023-11-08T13:37:53Z",
+      "data_transfer_bytes": 0,
+      "written_data_bytes": 6296,
+      "written_data_bytes_updated_at": "2023-11-08T13:37:53Z",
+      "compute_time_seconds": 708,
+      "compute_time_seconds_updated_at": "2023-11-07T19:43:17Z",
+      "active_time_seconds": 672,
+      "active_time_seconds_updated_at": "2023-11-07T19:43:17Z",
+      "updated_at": "2023-11-08T19:08:56Z",
+      "period_start": "2023-11-01T00:00:00Z",
+      "period_end": null,
+      "previous_period_id": "4abcae52-490c-4144-a657-ed93139e2b4e"
+    },
+    {
+      "project_id": "divine-tree-77657175",
+      "period_id": "f8f50267-69d2-4891-8359-847c138dbf80",
+      "data_storage_bytes_hour": 6109745400,
+      "data_storage_bytes_hour_updated_at": "2023-11-08T19:07:53Z",
+      "synthetic_storage_size": 32673288,
+      "synthetic_storage_size_updated_at": "2023-11-06T22:58:17Z",
+      "data_transfer_bytes": 0,
+      "written_data_bytes": 2256,
+      "written_data_bytes_updated_at": "2023-11-06T22:43:17Z",
+      "compute_time_seconds": 0,
+      "active_time_seconds": 0,
+      "updated_at": "2023-11-08T19:08:56Z",
+      "period_start": "2023-11-01T00:00:00Z",
+      "period_end": null,
+      "previous_period_id": "385be9ab-5d6c-493e-b77f-d8f28a5191ca"
+    }
+  ],
+  "periods_in_response": 2,
+  "pagination": {
+    "cursor": "divine-tree-77657175"
+  }
+}
+```
+</CodeBlock>
+</details>
+
+Key details:
+* The `period_id` key and `previous_period_id` are unique values used to identify and connect periods across the time range. 
+* The `period_start` and `period_end` keys show the dates for that particular billing period. A `null` value indicates that the object is for the current billing period.
+* The `cursor` object under `pagination` shows the last project Id in the response. See more about pagination in the next section.
+
+#### Control pagination for large result sets
+
+To control pagination (number of results per response), you can include these query parameters:
+* `limit` &#8212; sets the number of project objects to be included in the response
+* `cursor` &#8212; by default, the response uses the project `id` from the last project in the list as the `cursor` value (included in the `pagination` object at the end of the response). Generally, it is up to the application to collect and use this cursor value when setting up the next request.
+
+Here is an example `GET` request asking for the next 100 projects, starting with project id `divine-tree-77657175`:
+
```

Comment:
Use `<CodeBlock shouldWrap>` and `</CodeBlock>` to wrap this long example.

Reply:
A question for Atli or someone who has worked with partners and "quotas": Should this document cover these project limits as well:

- autoscaling_limit_min_cu
- autoscaling_limit_max_cu
- suspend_timeout_seconds
- history_retention_seconds